drv/vsc7448/src/lib.rs

```diff
@@ -25,6 +25,9 @@ pub use vsc_err::VscError;
 
 use crate::dev::Dev10g;
 
+/// Maximum port count
+pub const PORT_COUNT: usize = 53;
```

Comment:
I also have this constant [in `control-plane-agent`](https://github.com/oxidecomputer/hubris/blob/master/task/control-plane-agent/src/mgs_sidecar.rs#L33-L34); could we move it to `monorail-api`? (Or maybe control-plane-agent could depend on this crate to grab it?)

Reply:
Done!